### PR TITLE
HMAN-566: Adding in error description to reports

### PR DESCRIPTION
**Pull request by shahir-ali**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-566

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java [hunk 1]
```diff
@@ -20,6 +20,9 @@ public HearingRequestForCsv toHearingRequestForCsv(CaseHearingRequestEntity requ
                 requestEntity.getCaseUrlContextPath(),
                 requestEntity.getPublicCaseName()
             ));
+        if (isError == true) {
+            hearingRequestForCsv.setErrorDescription(requestEntity.getHearing().getErrorDescription().toString());
```
Comment (padmapriyanalam) [inline src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java:24]: boolean flag is not needed as AC01B in Jira says that  No Error Description values should be displayed as blank

File: src/main/java/uk/gov/hmcts/reform/hmc/service/OperationalReportsService.java [hunk 2]
```diff
@@ -13,11 +13,11 @@ public interface OperationalReportsService {
 
     File createCsvDataForAwaitingActuals() throws IOException;
 
-    List<HearingRequestForCsv> createCsvObjectsForGivenStatuses(List<String> statuses);
+    List<HearingRequestForCsv> createCsvObjectsForGivenStatuses(List<String> statuses, boolean isError);
```
Comment (padmapriyanalam) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/OperationalReportsService.java:16]: isError flag not needed and can be removed in all the files

File: src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java (hunk 1 above)
Comment (shahir-ali) [reply src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java]: Boolean flags have been removed in latest commit and no error description is displayed as blank as needed for AC01B

File: src/main/java/uk/gov/hmcts/reform/hmc/service/OperationalReportsService.java (hunk 2 above)
Comment (shahir-ali) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/OperationalReportsService.java]: isError flag has been removed in latest commit

**Review by padmapriyanalam — COMMENTED** [review]
Add unit test / integration test files please

File: src/integrationTest/resources/sql/add-error-description.sql
Comment (padmapriyanalam) [inline src/integrationTest/resources/sql/add-error-description.sql:1]: The .sql file is not used. This can be deleted

**Review by padmapriyanalam — APPROVED** [review]

**Pull request merged**